landoscript/src/landoscript/actions/android_l10n_import.py

```diff
@@ -100,6 +108,25 @@ async def run(
 
             diff += diff_contents(orig_file, new_file, l10n_file.dst_name)
 
+        for info in android_l10n_import_info.toml_info:
```

Comment:
nit: `for info in android_l10n_import_info.toml_info` here vs `for toml_info in android_l10n_import_info.toml_info` above; might be better to stick to the same variable name?

Reply:
Yeah, that's fair.